### ISSUE TITLE
[Question] Any Chance for Update to 4.8? :)

### ISSUE THREAD

**Issue by fuba82** [Phoenix09/filebot#9]
hi there,

is there any chance to Update to Version 4.8?
just asking, no stress ;)

**Comment by thiagorobis** [comment]
It will be really nice!

**Comment by neik1** [comment]
Agree! A little update would be really nice and appreciated by all of us! :-)

**Comment by barry-allen07** [comment]
Java 10 is needed for 4.8.2

https://bit.ly/2ISka9L (4.8.2 on Mega, feel free to upload to different mirrors)

Only Windows and Linux files for now. Mac OS will be added at a later time. 

Enjoy!!

**Comment by fuba82** [comment]
> Java 10 is needed for 4.8.2
> 
> https://bit.ly/2ISka9L (4.8.2 on Mega, feel free to upload to different mirrors)
> 
> Only Windows and Linux files for now. Mac OS will be added at a later time.
> 
> Enjoy!!

Seems to work!
Thanks dude! :)

**Comment by barry-allen07** [comment]
> Seems to work!
Thanks dude! :)

Not a problem :)

**Comment by deadspool** [comment]
Hi,
Thanks for your share barry-allen07.
Can you please tell me how can I install this version on a Red-Hat distrib (CentOS 7. I have already a working Filbot v. 4.7.9 and JRE 10).
Maybe copy/paste the Filebot.jar from Windows?

**Comment by macoloco** [comment]
4.8.2 is working well..JRE 10

**Comment by d4k0** [comment]
Hmm, quite a few virus scanners complain that the provided version 4.8.2 contains malware:

https://www.virustotal.com/#/file/c184b521b7ec47b362d810c4faf690c741f35e2391201500a3996ca1809222c3/detection
https://www.virustotal.com/#/file/44100d1ddf5cb1289ac8d92ccdef82ac570045bf3163deac4124f96f52df4e21/detection

Phoenix09's last version is "clean":

https://www.virustotal.com/#/file/c67307aed489efa68ac940404d257ce0f0ff7ef50c5eefa5bd25c0c231e8a7c1/detection

I'm currently trying to compile the latest FileBot version myself (but without any changes to the code) and see if it works. If this is successful, I may create my own repository and upload new versions there.

EDIT:
I compiled it and got the same exe files. I'm not sure what happened here ... Maybe it has something to do with the [new license funtion](https://www.filebot.net/forums/viewtopic.php?f=6&t=6006)?

**Comment by barry-allen07** [comment]
I only disabled the license checks on my compiled version. No other changes were made from the original repo. I guess the way Filebot checks for its license file could be detected as a virus. 

If you end up creating your own repo please post a link here. I might create my own repo once I have some free time available.

**Comment by viperdriver2000** [comment]
Here is a mirror of all three files in one zip https://www62.zippyshare.com/v/Wk4ArPgz/file.html
@d4k0 i use it in an "sandboxie" so i see no problems.

**Comment by d4k0** [comment]
@barry-allen07
> I only disabled the license checks on my compiled version. No other changes were made from the original repo.

May I ask what has to be changed to disable it? I only had a quick look at `License.java` and I'm not sure if it can be disabled there (it checks the license there, but it also seems to build some needed (?) objects) ... On the other hand the current version somehow already says that it is registered.

> I guess the way Filebot checks for its license file could be detected as a virus.

Well, the "malware" is said to be in the exe files which are built by rednoah and should only be there to launch Java, but these file are also much bigger compared to older versions.

> If you end up creating your own repo please post a link here. I might create my own repo once I have some free time available.

I will probably wait a little bit and see how the license thing works out (you already mentioned that it can be disabled) ... Creating new versions (at least for Windows) would be no problem for me as I just have to start a batch file. The only problem I had was that `log4j-api.jar` was not copied to the FileBot lib folder (although it is downloaded by Apache Ivy), but I "fixed" this by copying it.

**Comment by thiagorobis** [comment]
The Filebot command line keeps checking license... look:

`License Error: REGISTERED
 ________________________________________________
/ Please purchase a FileBot License:             \
\ https://www.filebot.net/purchase.html#msi      /
 ------------------------------------------------
      \                             .       .
       \                           / `.   .' "
        \                  .---.  <    > <    >  .---.
         \                 |    \  \ - ~ ~ - /  /    |
               _____          ..-~             ~-..-~
              |     |   \~~~\.'                    `./~~~/
             ---------   \__/                        \__/
            .'  O    \     /               /       \  "
           (_____,    `._.'               |         }  \/~~~/
            `----.          /       }     |        /    \__/
                  `-.      |       /      |       /      `. ,~~|
                      ~-.__|      /_ - ~ ^|      /- _      `..-'
                           |     /        |     /     ~-.     `-. _  _  _
                           |_____|        |_____|         ~ - . _ _ _ _ _>
FileBot requires a valid license. Please run `filebot --license *.psm` to install your FileBot license.`

**Comment by td75** [comment]
Thank you for sharing this barry-allen07!

Was wondering if there was a way to disable the pop-up showing there is an update available for same version?

**Comment by barry-allen07** [comment]
Hey everyone,

I`m back at it again! I have updated Filebot to the latest commit as of 8/22/2018.

Changelog:

*Fixed Filebot command line interface bugs
*Disabled updates checks (No more prompts to update)
*Added portable Linux file (Should work on any Linux distro)
*Fixed weird issues while installing .deb file
*Updated included Java in .deb and .msi files to 10.0.2

***At least Java 10.0.1 is needed if using portable files*** (No need to install Java if using .msi or .deb files)


Please post here if you find any issues or bugs

Enjoy!! (Link below)

https://bit.ly/2ISka9L

**Comment by NaruZosa** [comment]
Hi Barry, any chance of you setting up your own repo with the changes? Looks like Phoenix is getting a bit sick of us based on their change to the readme a couple of months ago:

"No, I will not be updating this. go read the license/readme in the original repo, then stop fucking asking."

**Comment by td75** [comment]
Thanks for the update barry-allen07:)

**Comment by sachatt** [comment]
Was unable to get this to work on windows 10. I installed JRE 10.0.2, however I get a "groovy.lang.MissingPropertyException: No such property: license for class: Script1" error message. Any advice/help ?

Note: I installed on Windows 10 with the MSI file and have Java 10.0.2

**Comment by macoloco** [comment]
Thanx Barry-allen07...FB 4.8.4 is working well

**Comment by barry-allen07** [comment]
Hello everyone!

I finally made my own repo with the changes detailed below. 
(Link Below)

*Fixed Filebot command line interface bugs
*Disabled updates checks (No more prompts to update)
*Added portable Linux file (Should work on any Linux distro)
*Fixed weird issues while installing .deb file
*Updated included Java in .deb and .msi files to 10.0.2

.Deb files are not up just yet. Those will be up during the next few days. 

Please direct all issues to the newly created repo. 

https://github.com/barry-allen07/FB-Mod

Thanks for using this version.

**Comment by sachatt** [comment]
Any chance of putting up the install files ?

Also do you know if there is a fix for the "groovy.lang.MissingPropertyException: No such property: license for class: Script1" error.

**Comment by NaruZosa** [comment]
They're available at the releases page.

https://github.com/barry-allen07/FB-Mod/releases


Also, someone else had the same issue but closed the issue without stating the resolution. https://github.com/barry-allen07/FB-Mod/issues/1

**Comment by Sada-Mio** [comment]
@BradleyDS2 

https://www.filebot.net/forums/viewtopic.php?t=6112

groovy.lang.MissingPropertyException: No such property: license for class: Script1 error from using a pre-release older version of filebot. However, your compiled version shows up as r5849, wonder if the source code is omitting something key we don't have.

**Comment by sachatt** [comment]
@BradleyDS2 

Any update on this ?

**Comment by NaruZosa** [comment]
I'm not the person working on it, @barry-allen07 is. Looks like it was fixed in the most recent release though.

**Comment by barry-allen07** [comment]
@torqueNoFriction This should be fixed on the latest release. Please see Scripts.md on how to use scripts.

Thanks for the feedback.
